test_unstructured_ingest/check-diff-evaluation-metrics.sh

```diff
@@ -29,7 +29,7 @@ trap cleanup EXIT
 # to update ingest test fixtures, run scripts/ingest-test-fixtures-update.sh on x86_64
 if [ "$OVERWRITE_FIXTURES" != "false" ]; then
     # force copy (overwrite) files from metrics-tmp (new eval metrics) to metrics (old eval metrics)
-    cp -rf "$EXPECTED_OUTPUT_DIR" "$EXPECTED_OUTPUT"
+    cp -rf "$OUTPUT_DIR" "$EXPECTED_OUTPUT_DIR"
```

Comment:
these vars could be more clear: maybe `EXPECTED_OUTPUT_DIR` -> `METRICS_DIR`  AND `OUTPUT_DIR` -> `TMP_METRICS_LATEST_RUN_DIR`?

Reply:
Renamed!